src/main/flight/pid.c

```diff
@@ -1245,7 +1245,10 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         }
 
         pidData[axis].S = getSterm(axis, pidProfile);
-        applySpa(axis, pidProfile);
+
+        if (!pidRuntime.axisInAngleMode[axis]) {
+            applySpa(axis, pidProfile);
+        }
```

Comment:
Add a `#ifdef USE_WING` block here. IMO this code should be isolated to optimize other use cases (RACING).